md/define-and-deploy-the-bdm.md

```diff
@@ -118,6 +118,8 @@ When you create a process that uses a business object with a composition or aggr
 * You can use getter and setter methods in process or activity operations to set the composed objects as a single instance or as a `java.util.list` for a list of instances.
 * If your process uses a query from a Groovy expression or uses DAO objects from a client Java application, load the complete composite objects including the child objects.
 
+<a id="lazy_eager_loading" />
```

Comment:
This add an anchor which is used in a reference in the new "bdm in rest api how to" page